pandas/tests/indexes/multi/test_monotonic.py

```diff
@@ -174,3 +174,9 @@ def test_is_strictly_monotonic_decreasing():
     )
     assert idx.is_monotonic_decreasing is True
     assert idx._is_strictly_monotonic_decreasing is False
+
+
+def test_is_monotonic_with_nans():
+    # GH: 37220
+    idx = pd.MultiIndex.from_tuples([(np.nan,), (1,)], names=["test"])
+    assert idx.is_monotonic_increasing is False
```

Comment:
Maybe parametrize over different positions for ``NaN``? I think it's probably correct to always return False here, although this is interesting:
```python
[ins] In [32]: idx
Out[32]: Float64Index([nan, 1.0, 2.0], dtype='float64')

[ins] In [33]: idx.sort_values(ascending=True).is_monotonic_increasing
Out[33]: False
```

Reply:
That really is interesting. With 0.25.3
``pd.MultiIndex.from_tuples([(1,), (np.nan,)], names=["test"]).is_monotonic_increasing`` returned ``True``. We could change that, but that would be more than a Regression fix then?

Reply:
I think probably fine to call it a regression fix. I would say the previous behavior is not correct in this case (if monotone increasing means `x <= y` for every adjacent pair of elements, then no sequence with ``NaN`` values is monotone increasing, or decreasing for that matter), so we'd only want to fix the case with ``NaN`` at the beginning rather than restoring the behavior completely.

Reply:
Thanks very much for the explanation. Adjusted the code accordingly.